src/pages/workspace/WorkspacesListRow.tsx

```diff
@@ -0,0 +1,168 @@
+                        iconTooltip={undefined}
+                        icon={undefined}
+                        iconFill={undefined}
+                        iconStyles={undefined}
+                        onIconPress={undefined}
+                        anchorAlignment={undefined}
+                        shouldOverlay={undefined}
+                        shouldSetModalVisibility={undefined}
+                        disabled={undefined}
+                    />
+                )}
+            </View>
+            <View style={[styles.flexRow, isWide && styles.flex1, styles.gap2, !isWide && styles.mr5, styles.alignItemsCenter]}>
+                <Avatar
+                    source={ownerDetails.avatar}
+                    size={CONST.AVATAR_SIZE.SMALL}
+                    containerStyles={styles.workspaceOwnerAvatarWrapper}
+                />
+                <View style={styles.flex1}>
+                    <Text
+                        numberOfLines={1}
+                        style={[styles.labelStrong]}
+                    >
+                        {PersonalDetailsUtils.getDisplayNameOrDefault(ownerDetails, 'displayName')}
+                    </Text>
+                    <Text
+                        numberOfLines={1}
+                        style={[styles.textMicro, styles.textSupporting]}
+                    >
+                        {ownerDetails.login}
+                    </Text>
+                </View>
+            </View>
+            <View style={[styles.flexRow, isWide && styles.flex1, styles.gap2, !isWide && styles.mr5, styles.alignItemsCenter]}>
+                <Icon
+                    src={workspaceTypeIcon(workspaceType)}
+                    width={34}
+                    height={34}
+                    additionalStyles={styles.workspaceTypeWrapper}
+                />
+                <View style={styles.dFlex}>
+                    <Text
+                        numberOfLines={1}
+                        style={styles.labelStrong}
+                    >
+                        {userFriendlyWorkspaceType}
+                    </Text>
+                    <Text
+                        numberOfLines={1}
+                        style={[styles.textMicro, styles.textSupporting]}
+                    >
+                        {translate('workspace.common.plan')}
+                    </Text>
+                </View>
+            </View>
+            {isWide && (
+                <ThreeDotsMenu
+                    menuItems={[]}
+                    anchorPosition={{top: 0, right: 0}}
+                    iconStyles={[styles.mr2]}
+                    iconTooltip={undefined}
+                    icon={undefined}
+                    iconFill={undefined}
+                    onIconPress={undefined}
+                    anchorAlignment={undefined}
+                    shouldOverlay={undefined}
+                    shouldSetModalVisibility={undefined}
+                    disabled={undefined}
```

Comment:
same comment as [this one](https://github.com/Expensify/App/pull/32817/files#r1428560147)